config/environments/production.rb

```diff
@@ -59,7 +59,20 @@
   # Use a real queuing backend for Active Job (and separate queues per environment).
   # config.active_job.queue_adapter     = :resque
   # config.active_job.queue_name_prefix = "unnamedrailsproject_production"
-
+  config.action_mailer.default_url_options = { host: "tokenowo-production.herokuapp.com" }
```

Comment:
We need it to work on staging as well

Reply:
On local machines we can use https://github.com/fgrehm/letter_opener_web to preview sending email.